back/app/models/custom_page.rb

```diff
@@ -29,9 +30,12 @@
 #
 # Indexes
 #
+#  index_custom_pages_on_code  (code)
 #  index_custom_pages_on_slug  (slug) UNIQUE
 #
 class CustomPage < ApplicationRecord
+  CODES = %w[about terms-and-conditions privacy-policy faq proposals custom].freeze
```

Comment:
This should probably be removed, in favor of the simpler validation: `... inclusion: %w[custom]`

Agree?